bnelson777/id.ly
# IL-124: Resend Message from MessageThread

# New Feature:
Users are now able to resend messages from the message thread by long pressing (press and holding) a message bubble.

# Testing:
1. Open up the message thread of any message from the inbox.
2. Long press (press and hold on) a message you wish to resend.
3. Confirm that a lockbox screen pops up with data
4. Send out that message
5. Retrieve message from email and decrypt to confirm the message is correct.
